WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/ranges/data/AnalyticsHubTodayRangeData.kt

```diff
@@ -0,0 +1,40 @@
+package com.woocommerce.android.ui.analytics.ranges.data
+
+import com.woocommerce.android.extensions.startOfCurrentDay
+import com.woocommerce.android.extensions.theDayBeforeIt
+import com.woocommerce.android.ui.analytics.ranges.AnalyticsHubTimeRange
+import com.woocommerce.android.ui.analytics.ranges.AnalyticsHubTimeRangeData
+import java.util.Calendar
+import java.util.Date
+
+// Responsible for defining two ranges of data, one starting from the the first second of the current day
+// until the same day in the current time and the previous one, starting from the first second of
```

Comment:
Is that right? I'm not sure what's "(...)day in the current time(...)"

```suggestion
// until the same day in the current timezone and the previous day, starting from the first second of
```